src/index.css

```diff
@@ -0,0 +1,21 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+
+* {
+    font-family: 'Jura', sans-serif;
+    color: #EAEAEA;
+}
+
+@layer components {
+    .btn-primary {
+      @apply w-full  border-2 border-green rounded-md py-1 bg-green text-black mb-4 hover:cursor-pointer hover:bg-green/85 font-semibold;
+    }
+    .input-area{
+        @apply w-full rounded-md pt-2 pl-4 pb-2 text-black focus:outline-none focus:ring focus:ring-blue focus:border-transparent;
```

Comment:
Let's apply a _text-blue_ or _text-white_ class for better readability

` @apply w-full rounded-md pt-2 pl-4 pb-2 text-black focus:outline-none focus:ring focus:ring-blue focus:border-transparent;`

Before:
![image](https://github.com/Code-the-Dream-School/ffprac-team1-front/assets/42615340/5f919dd0-0d2b-4670-98f1-c466faa91742)

After:
![image](https://github.com/Code-the-Dream-School/ffprac-team1-front/assets/42615340/fe805ef7-fc61-4a4c-9581-eeebd6d07558)